outputs.tf

```diff
@@ -26,3 +26,8 @@ output "kafka_http_url" {
   description = "The API endpoint to interact with Event Streams REST API"
   value       = ibm_resource_instance.es_instance.extensions.kafka_http_url
 }
+
+output "kafka_broker_version" {
+  description = "The Kafka version"
+  value       = lookup(ibm_resource_instance.es_instance.extensions, "kafka_broker_version", "")
```

Comment:
Is there a need to do lookup here?

Reply:
Not a need but just a safety measure to fall against a default value.